packages/api/mocks/TopSearchesQuery.ts

```diff
@@ -0,0 +1,16 @@
+export const TopSearchesQuery = `query TopSearchesQuery {
```

Comment:
Remember to remove this

Reply:
Why? 🤔 This is the mocked requests for the tests to run